modules/webform_event_dispatcher/webform_event_dispatcher.info.yml

```diff
@@ -0,0 +1,7 @@
+name: Webform Event Dispatcher
+type: module
+description: Registers event dispatchers for several webform hooks
+core: 8.x
+package: Development
+dependencies:
+  - hook_event_dispatcher:hook_event_dispatcher
```

Comment:
Shouldn't we add a dependency to webform? Makes no sense to enable this module when webform isn't enabled?

Reply:
Correct! 🤦‍♂️ Adding dependency.